admins/pageflow/entry.rb

```diff
@@ -49,7 +49,9 @@ module Pageflow
     end
 
     filter :title
-    filter :account
+    filter :account,
+           if: ->(_) { authorized?(:index, :accounts) },
+           collection: -> { Account.accessible_by(current_ability, :see_in_list) }
```

Comment:
While `can :read, Account` maps to `#resolve`, it also has a block. If we wanted to use it here, we could for that reason not use `accessible_by`. Additionally, `read?` in the account policy aliases `manage?`, which might require further changes or could at least lead to misunderstandings, since we do not want to show only accounts with at least `manage`-Permissions here.

The Hound warning should by now be obsolete, since `account` is actually used in the line below the one mentioned in the warning.